### DISCUSSION TITLE
Enhancement: Smooth transition of the health bar.

### DISCUSSION THREAD

**Discussion by SinexProgs** [FunkinCrew/Funkin#1628] [Ideas]
After the new update, there is now no transition between songs in the week. And now you can see how the health bar resets without any animation.
I suggest that you either make the points and health count smoothly from the last song.
![health_bar_transition1](https://user-images.githubusercontent.com/52968560/113460552-5aadee80-9443-11eb-9224-f3c66ef3fbbb.gif)
Or if this is difficult for you, just make it so that before and after the song, the health bar and points are moved out from the bottom.
![health_bar_transition2](https://user-images.githubusercontent.com/52968560/113460663-b2e4f080-9443-11eb-8b0c-6bf0ec09577a.gif)
That would be nice.

**Comment by amyspark-ng** [comment]
this would look awesome
